applications/asset_tracker_v2/sample.yaml

```diff
@@ -121,11 +121,11 @@ tests:
   applications.asset_tracker_v2.carrier:
     build_only: true
     build_on_all: true
-    platform_allow: nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns thingy91_nrf9160_ns
     integration_platforms:
       - nrf9160dk_nrf9160_ns
+      - thingy91_nrf9160_ns
     extra_args: OVERLAY_CONFIG=overlay-carrier.conf
-                DTC_OVERLAY_FILE=nrf9160dk_with_ext_flash.overlay
```

Comment:
@kacperradoszewski ATv2 + carrier now builds again with the optimizations brought by picolibc. I'm assuming that is why external flash was added to this build?

Reply:
Yes, this change was to make it fit in CI, fine to remove it if it builds again. Nice that the Thingy build works too 👍